### PR TITLE
Fix RessourceId for Datasets

### PR DESCRIPTION
**Pull request by jdroenner**
i also added a convenience method (by accident) "get_items_by_name". However, it is so useful lets keep it.

### CODE REVIEW

**Comment by jdroenner** [comment]
[Permissions können nicht für Datasets vergeben werden.](https://trello.com/c/DdAoz0aR/1277-permissions-k%C3%B6nnen-nicht-f%C3%BCr-datasets-vergeben-werden)

File: geoengine/layers.py [hunk 1]
```diff
@@ -394,6 +394,11 @@ def add_existing_collection(self,
 
         return collection_id
 
+    def get_items_by_name(self, name: str) -> List[Listing]:
+        '''Get all children with the given name'''
+
+        return [item for item in self.items if item.name == name]
```
Comment (ChristianBeilschmidt) [inline geoengine/layers.py:400]: Can there by more than one item with that name?
Could there be a wildcard, like `'name.*"?

And…could you then update https://github.com/geo-engine/geoengine-python/blob/main/examples/layers.ipynb?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: geoengine/layers.py (hunk 1 above)
Comment (jdroenner) [reply geoengine/layers.py:400]: i can add 2 layers with the same name so i guess that it is possible. I had no need for a wildcard yet. So i would suggest we add that when it is needed.

Comment (jdroenner) [reply geoengine/layers.py:400]: [d3bc79b](https://github.com/geo-engine/geoengine-python/pull/118/commits/d3bc79b55b3276c19f7dcde240bce3cf8757465f)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**